File: app/src/main/java/com/kyanro/ltdemo/MainActivity.java
package com.kyanro.ltdemo;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

import lombok.Getter;
import lombok.Setter;

public class MainActivity extends AppCompatActivity {

    @SuppressLint("SetTextI18n")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView hello = (TextView) findViewById(R.id.hello_text);

        MyData myData = new MyData();

        myData.setHoge("Hello");
        myData.setFuga("World");

        hello.setText(myData.getHoge() + " " + myData.getFuga());
    }

    static class MyData {
        @Getter
        @Setter
        private String hoge;
        @Getter
        @Setter
        private String fuga;
    }
}

File: app/src/main/java/com/kyanro/ltdemo/MainActivity.kt
package com.kyanro.ltdemo

import android.content.Intent
import android.databinding.DataBindingUtil
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.kyanro.ltdemo.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = DataBindingUtil.setContentView<ActivityMainBinding>(this, R.layout.activity_main)
        binding.body.setOnClickListener {
            val intent: Intent = Intent(this, SubActivity::class.java)
            startActivity(intent)
        }

        val container = Container()

        // setter getter が1つずつある////
        container.data = "MainActivity!"
        binding.body.text = container.data
        //////////////////////////////////
    }
}
